app/controllers/incoming_webhooks/mailchimp_unsubscribes_controller.rb

```diff
@@ -0,0 +1,30 @@
+class IncomingWebhooks::MailchimpUnsubscribesController < ApplicationController
+  class InvalidListID < StandardError; end
+
+  LIST_MAPPINGS = {
+    mailchimp_newsletter_id: :email_newsletter,
+    mailchimp_sustaining_members_id: :email_membership_newsletter,
+    mailchimp_tag_moderators_id: :email_tag_mod_newsletter,
+    mailchimp_community_moderators_id: :email_community_mod_newsletter
+  }.freeze
+
+  def create
+    not_authorized unless valid_secret?
+    user = User.find_by!(email: params.dig(:data, :email))
+    user.update(email_type => false)
```

Comment:
```suggestion
    user.update(email_type: false)
```

we should be able to use the newer syntax here, right?

Reply:
I think that wouldn't work here because `email_type` is evoking the method on line 23.

Reply:
Ah, whoops, totally misread that. You're right!